website/layouts/index.redirects

```diff
@@ -30,3 +30,4 @@
 /docs/reference/env_vars   /docs/reference/cli/#environment-variables  301
 /docs/usage/*   /docs/administration/:splat  301
 /docs/setup/installation/package-managers/vector-installer   /docs/setup/installation/manual/vector-installer  301
+/docs/reference/configuration/sources/generator /docs/reference/configuration/transforms/demo_logs 301
```

Comment:
@lucperkins if you get a sec, I could probably use your 👀 here. I added this redirect to redirect the old name, `generator`, to the new name, `demo_logs`, but the link checker seems to not like this:

https://app.netlify.com/sites/vector-project/deploys/61927bcae40c610007c0f9bc#L143

Does anything jump out to you?